presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -93,4 +96,33 @@ public GlueHiveMetastoreConfig setCatalogId(String catalogId)
         this.catalogId = Optional.ofNullable(catalogId);
         return this;
     }
+
+    @Min(1)
+    @Max(10)
+    public int getPartitionSegments()
+    {
+        return partitionSegments;
+    }
+
+    @Config("hive.metastore.glue.partitions-segments")
+    @ConfigDescription("Number of segments for partitioned Glue tables")
+    public GlueHiveMetastoreConfig setPartitionSegments(int partitionSegments)
+    {
+        this.partitionSegments = partitionSegments;
+        return this;
+    }
+
+    @Min(1)
+    public int getGetPartitionThreads()
```

Comment:
nit: `getGetPartitionThreads`. Two issue:

1. It should be an noun for get/set method (e.g. partitionFetchingThreadCount)
2. We should add `glue` into the method name to reflect it only affects Glue related metastore :) 

What about `getGluePartitionFetchingThreadCount`?  ditto for the set method.

Reply:
I hear you, but I feel that it would be inconsistent with the rest of the code base. [Example](https://github.com/prestodb/presto/blob/1d323240ef5628fb5fc05ebb866bc47bf218e888/presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java#L136).  Also we should not need to add the "Glue" prefix since the class is specific to Glue.

Reply:
@anoopj : But isn't this configuration itself tied to Glue?  

We should decide what the convention is, and if necessary, fix other use cases. What do you think whether we want "Glue" in method name ? @highker , @arhimondr